spark/src/main/scala/geotrellis/spark/io/accumulo/spacetime/SpaceTimeRasterRDDWriter.scala

```diff
@@ -34,7 +35,7 @@ object SpaceTimeRasterRDDWriter extends RasterRDDWriter[SpaceTimeKey] {
       .reduce { boundable.combine }
   }
 
-  /** TODO: What is the rules about the "num" parameter? */
+  /** TODO: What are the rules about the "num" parameter? */
```

Comment:
'num' should be at least as large as the number of tablet servers you have. If they are some small multiple (2x, 3x) it is also ok. Number of splits determines the number of empty tablets and number of concurrent connections that can be opened for ingest to an empty table.